Source/JavaScriptCore/inspector/InjectedScriptSource.js

```diff
@@ -223,22 +225,37 @@ let InjectedScript = class InjectedScript extends PrototypelessObjectBase
                 try {
                     resolvedArgs[i] = this._resolveCallArgument(callArgs[i]);
                 } catch (e) {
-                    return @String(e);
+                    callback(this._createThrownValue(e, objectGroupName));
```

Comment:
ditto for protocol command error behavior change

Reply:
Done.